README.md

```diff
@@ -369,6 +388,30 @@ The `iotedgedev` module has the following commands:
 `iotedgedev project --help`
 - `--create TEXT`  Creates a new Azure IoT Edge project. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.
 
+
+**azure**
+
+`iotedgedev azure --help`
+
+- `--setup`                    Reads the required Azure components
+                               configuration from your subscription. Creates
+                               new Azure resources or uses existing ones.
+                               [required]
+- `--azure-credentials TEXT TEXT`  The credentials (username password) to use to
```

Comment:
To me, it is not intuitive to say leave empty to use interactive mode. How about this

--credentials username password

if --credentials not specified, then interactive.